routemaster-client.gemspec

```diff
@@ -16,7 +16,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^spec/})
   spec.require_paths = %w(.)
 
+  spec.add_runtime_dependency     'typhoeus', '~> 1'
   spec.add_runtime_dependency     'faraday', '>= 0.9.0'
-  spec.add_runtime_dependency     'net-http-persistent', '>= 2.9.4'
   spec.add_runtime_dependency     'wisper', '>= 1.6.1'
+  spec.add_runtime_dependency     'oj'
```

Comment:
Considering that this is a library, can use a permissive lock on the oj version? Just to make sure that different apps deal with the JSON payloads the same way.